prelude/toolchains/cxx.bzl

```diff
@@ -67,7 +67,6 @@ def _system_cxx_toolchain_impl(ctx: AnalysisContext):
         shared_library_name_default_prefix = ""
         shared_library_name_format = "{}.dll"
         shared_library_versioned_name_format = "{}.dll"
-        additional_linker_flags = ["msvcrt.lib"]
```

Comment:
This one was added to make hello_world example to work on Windows. Do you know how does it build now?

Reply:
I can't think of any good reason. I tried to reproduce a failure like that with reasonable flags but couldn't get that error. Like it says in the documentation https://learn.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-170 the compiler places a linking directive in the object file specifying the runtime library to use (by default MT_StaticRelease on my machine).

If you pass in this flag https://learn.microsoft.com/en-us/cpp/build/reference/zl-omit-default-library-name?view=msvc-170 you do get that error but I did a CTRL+F through the repository and didn't find that flag anywhere.